tenant/sunbird/index.html

```diff
@@ -128,7 +128,8 @@ <h1 class="sb-text-upper mt-0">Learning Platform</h1>
                             <h2 class="sb-text-upper">National Urban</h2>
                             <h1 class="sb-text-upper mt-0">Learning Platform</h1>
                             <p class="sb-font">Launching SBM 2.0 with the vision to make ‘Garbage Free Cities’, <br> which aims to sustain and advance the progress in urban <br> sanitation and waste management</p>
-                            <div><a class="sb-btn-default mt-10" href="https://nulp.niua.org/resources?SBMRegistration">Sign In</a>
+                            <div>
```

Comment:
@shreyas1434shinde  I'll not be going to merge this command til the title is not in the proper format

Reply:
@komalm updated